docs/installation.rst

```diff
@@ -63,6 +63,22 @@ Once installed, you can start a test server::
 
 And point your browser at `http://localhost:5000 <http://localhost:5000>`_.
 
+Configure database with MySQL/MariaDB (optional)
+================================================
+
+Only required if you prefer MySQL/MariaDB over SQLite.
+
+1. Install PyMySQL dependencies. On Debian or Ubuntu, that would be::
+
+    apt install python3-dev libssl-dev
+
+2. Install PyMySQL (within your virtualenv)::
+
+    pip install 'PyMySQL>=0.9,<0.10'
+
+3. Create an empty database and a database user
```

Comment:
That might be a good idea to give instructions here on how to do so, so the instructions are completely noob-proof (And they will help me, too ;))

Reply:
Well… It would help me too :-p.

There are different ways to create a MySQL database, I do use [yunohost helpers](https://github.com/YunoHost-Apps/ihatemoney_ynh/blob/master/scripts/install#L35) for that. 

Some people may use phpmyadmin or some other frontend.

All that to say I won't write that myself and now, as I do not have anything ready to copy-paste.

So IMHO, we could merge without further details (better than nothing), and open a ticket to remember providing detailed instructions with MySQL Command line.